UnitySDK/Assets/ML-Agents/Examples/SharedAssets/Materials/PushBlockGround.physicMaterial

```diff
@@ -0,0 +1,13 @@
+%YAML 1.1
```

Comment:
Should this file really be in the SharedAssets folder ? It is called PushBlockGround so I would put in the PushBlock folder...

Reply:
I actually don't think this file is used anywhere.